### DISCUSSION TITLE
Names for theorem-like blocks confuse VS Code

### DISCUSSION THREAD

**Discussion by tobydriscoll** [quarto-dev/quarto-cli#7144] [Feature Requests]
### Description

This has become an annoyance for me. If you give a theorem-like block a name, you use a heading hashtag, as in:

```{markdown}
:::{#thm-euler}
# Euler's theorem
What a great theorem.
::: 
```
That works fine, but VS Code sees this as a top-level header and tries to organize the breadcrumbs accordingly, which ends up being confusing. It's not clear to me whether the quarto extension can fix this, or whether the syntax is too much of a clash.

**Comment by mcanouil** [comment]
FYI, this is tracked in the proper repository, see:
- https://github.com/quarto-dev/quarto/issues/294
